utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -83,8 +84,19 @@ pub struct StorageParams {
 	#[clap(long)]
 	pub skip_write: bool,
 
+	/// Specify the Handlebars template to use for outputting benchmark results.
+	#[clap(long)]
+	pub template_path: Option<PathBuf>,
+
+	/// Write the raw 'read' results in JSON format to the specified file.
+	#[clap(long)]
+	pub json_read_path: Option<String>,
```

Comment:
Should be of type `PathBuf`.

Reply:
This may be out of scope, but should we do the same [here](https://github.com/paritytech/substrate/blob/5afd5b8a4ee7bfb7d60d114b0968910f12748160/utils/frame/benchmarking-cli/src/storage/cmd.rs#L62) while we are at it?

Reply:
Yea sure if it does not cause any additional trouble for you :+1: